src/Microsoft.Dnx.Tooling/Utils/NuGetPackageUtils.cs

```diff
@@ -77,23 +77,21 @@ internal static class NuGetPackageUtils
                     // Fixup the casing of the nuspec on disk to match what we expect
                     var nuspecFile = Directory.EnumerateFiles(targetPath, "*" + NuGet.Constants.ManifestExtension).Single();
 
-                    if (!string.Equals(nuspecFile, targetNuspec, StringComparison.Ordinal))
```

Comment:
Have you considered doing IgnoreCase check *and* printing the warning if the names still don't match? (I don't feel very strongly about about that though...)